core/shared/src/main/scala/org/polyvariant/colorize/custom/ConfiguredColorize.scala

```diff
@@ -25,6 +26,8 @@ class ConfiguredColorize(config: RenderConfig) {
 
   implicit final def liftStringToColored(s: String): ColorizedString = ColorizedString.wrap(s)
```

Comment:
Can we remove this and replace it with a `Colorize[String]` in the companion object of `Colorize`?